utils/sql/git/required/2023_02_28_npc_scaling_zone_list_version_list.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `npc_scale_global_base`
+    CHANGE COLUMN `zone_id` `zone_id_list` text CHARACTER SET latin1 COLLATE latin1_swedish_ci NOT NULL DEFAULT 0 AFTER `level`,
+    CHANGE COLUMN `instance_version` `instance_version_list` text CHARACTER SET latin1 COLLATE latin1_swedish_ci NOT NULL DEFAULT 0 AFTER `zone_id_list`,
+    DROP PRIMARY KEY,
+    ADD PRIMARY KEY (`type`, `level`, `zone_id_list`(255), `instance_version_list`(255)) USING BTREE;
```

Comment:
Since we're overloading `zone_id_list` and `instance_version_list` here - it probably doesn't make sense to make this part of the primary key